tests/devices/unit_tests/test_bimorph_mirror.py

```diff
@@ -29,20 +30,35 @@ def valid_bimorph_values(mirror: BimorphMirror) -> dict[int, float]:
 
 
 @pytest.fixture
-def mock_vtrgt_vout_propogation(mirror: BimorphMirror):
+def bimorph_functionality(mirror: BimorphMirror):
```

Comment:
Should: I would suggest this instead

```python
@pytest.fixture
def mirror_with_mocked_put(mirror: BimorphMirror) -> BimorphMirror:
    # Do things to mirror
    return mirror
```

Then depend on `mirror_with_mocked_put` _instead of_ `mirror` in tests which need this logic. It reads better.